src/client.js

```diff
@@ -499,7 +499,10 @@ Client.prototype.quit = function() {
 			const socket = sockets.connected[user];
 
 			if (socket) {
-				socket.emit("sign-out");
+				if (signOut) {
```

Comment:
This feels like an unrelated fix.

Reply:
It's not, we don't want to sign out users on server close.

Reply:
Oh woops, I missed that we were already sending sign-out. Sounds good.